test/puppetlabs/ssl_utils/core_test.clj

```diff
@@ -618,6 +618,44 @@
                    (open-ssl-file "ca_crl.pem"))]
       (is (instance? SSLContext result)))))
 
+(deftest generate-ssl-context-test
+  (let [base-path   "test-resources/puppetlabs/ssl_utils/examples/ssl/"
```

Comment:
We have a utility function called `open-ssl-file` I think for handling this part so you don't need to repeat it in every test